app/src/components/Modals/activity-modal/activity-modal-body.tsx

```diff
@@ -233,7 +222,7 @@ const ActivityModalBody = ({
                 {f.options && (
                   <FormControl key={idx} className="w-full">
                     <BuildingTypeSelectInput
-                      options={f.options}
+                      options={f.options as string[]}
```

Comment:
I think we need to map or change the data type of this property here if the data is in a different format now?